tests/clients/cli/wrapper.go

```diff
@@ -281,6 +281,65 @@ func (w *Wrapper) GetAllCollections(ctx context.Context) ([]client.Collection, e
 	return cols, err
 }
 
+func (w *Wrapper) GetSchemaByName(ctx context.Context, name string) ([]client.SchemaDescription, error) {
+	args := []string{"client", "schema", "describe"}
+	args = append(args, "--name", name)
+
+	data, err := w.cmd.execute(ctx, args)
+	if err != nil {
+		return nil, err
+	}
+	var schema []client.SchemaDescription
+	if err := json.Unmarshal(data, &schema); err != nil {
+		return nil, err
+	}
+	return schema, err
+}
+
+func (w *Wrapper) GetSchemaByVersionID(ctx context.Context, versionID string) (client.SchemaDescription, error) {
+	args := []string{"client", "schema", "describe"}
+	args = append(args, "--version", versionID)
+
+	data, err := w.cmd.execute(ctx, args)
+	if err != nil {
+		return client.SchemaDescription{}, err
+	}
+	var schema client.SchemaDescription
+	if err := json.Unmarshal(data, &schema); err != nil {
+		return client.SchemaDescription{}, err
+	}
+	return schema, err
+}
+
+func (w *Wrapper) GetSchemaByRoot(ctx context.Context, root string) ([]client.SchemaDescription, error) {
+	args := []string{"client", "schema", "describe"}
+	args = append(args, "--root", root)
+
+	data, err := w.cmd.execute(ctx, args)
+	if err != nil {
+		return nil, err
+	}
+	var schema []client.SchemaDescription
+	if err := json.Unmarshal(data, &schema); err != nil {
+		return nil, err
+	}
+	return schema, err
+}
+
+func (w *Wrapper) GetAllSchema(ctx context.Context) ([]client.SchemaDescription, error) {
+	args := []string{"client", "schema", "describe"}
```

Comment:
suggestion: 3 of these methods have almost identical bodies. They differ only by `args`. 

It will look cleaner if you extract the common code and pass only `args` there

Reply:
I'm not sure that will be cleaner, as these are very simple functions that have no reason to be coupled via a private function, atm it is quite easy to see what they do, and a private func may damage that.

- [ ] Consider private func

Reply:
I had a look, and I prefer the current setup.  Leaving as is.